roles/openshift_sdn/files/sdn-ovs.yaml

```diff
@@ -83,10 +83,10 @@ spec:
         resources:
           requests:
             cpu: 100m
-            memory: 200Mi
```

Comment:
Why bump request?  What's the floor for a 10 node 100 pod cluster?

Reply:
58MB in my tests